buildspec/windowsTests.yml

```diff
@@ -14,7 +14,10 @@ env:
 phases:
   install:
     commands:
-      - choco install -y --no-progress codecov
+      - |
+        if(-Not($Env:CODEBUILD_BUILD_SUCCEEDING -eq "0" -Or $Env:CODE_COV_TOKEN -eq $null)) {
```

Comment:
Since this is the first command in the phase, isn't `CODEBUILD_BUILD_SUCCEEDING` always true?

Reply:
I wasn't sure where that env var actually gets defined. Will remove it from here.